FixMatrix/FixMatrix.cpp

```diff
@@ -0,0 +1,208 @@
+#include <cstdlib>
+#include <iostream>
+#include <stdexcept>
+
+namespace {
+
+    class matrix {
+        int Row, Col;
+        double **Value;
+    public:
+
+        matrix(unsigned int, unsigned int);
+
+        matrix(const matrix &);
+
+        int GetRow();
+
+        int GetCol();
+
+
+        friend bool operator==(const matrix &m1, const matrix &m2);
+
+        friend bool operator!=(const matrix &m1, const matrix &m2);
+
+        friend std::istream &operator>>(std::istream &istr, matrix &m);
+
+        friend std::ostream &operator<<(std::ostream &ostr, matrix &m);
+
+        friend matrix operator+(const matrix &m1, const matrix &m2);
+
+        friend matrix operator-(const matrix &m1, const matrix &m2);
+
+        friend matrix operator*(const matrix &m1, const matrix &m2);
+    };
+
+
+    bool operator==(const matrix &m1, const matrix &m2) {
+        return m1.Row == m2.Row && m1.Col == m2.Col;
+    }
+
+    bool operator!=(const matrix &m1, const matrix &m2) {
+        return !(m1 == m2);
+
+    }
+
+
+    ~
+
+    matrix();
+};
+
+matrix::matrix(int row, int col) {
+    Row = row;
+    Col = col;
+    Value = new double *[row];
+    for (int i = 0; i < row; i++) Value[i] = new double[col];
```

Comment:
Always put the body of "if", "for", etc. in the curly brackets block, even if it's a single line. For example:
```
if (flag) {
  do_something();
}
```